resources/views/emails/claim.blade.php

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>Claim locker {{ $lockerid }}</title>
```

Comment:
In the future, we should change this to the locker position. A locker 'GUID' is something that is generally irrelevant for end users.

Reply:
Changed.